### PR TITLE
159: Markdown-to-HTML

### PR DESCRIPTION
**Pull request by astolfivincent**
Steps to Test
-Go to blog listing page
-Data is pulled in via gray matter

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,50 +1,62 @@
...
 var fs = require('fs');
+var matter = require('gray-matter');
 
-fs.readFile('api.json', 'utf-8', function (error, data) {
-    var blogData = JSON.parse(data);
-    for(var blog in blogData) {
-        var Blog = blogData[blog];
+fs.readdir('blogs', function (err, files) {
+    var listingarr = [];
+    var count = 0;
```
Comment (rcaracaus) [inline app.js]: can't you just slice the array instead of counting?

Comment (rcaracaus) [inline app.js]: Or the forEach function should give you access to the index?

**Comment by rcaracaus** [comment]
About page looks destroyed

**Comment by rcaracaus** [comment]
https://desarol-pr-160.herokuapp.com/about/

**Comment by rcaracaus** [comment]
Now the blog detail pages aren't working.  https://desarol-pr-160.herokuapp.com/blog/2/

**Comment by astolfivincent** [comment]
This PR doesn't have the blog detail code in it. That's in the blog detail style pr. I assure you it's working, I've even merged this branch into the blog detail style one and the listings are appearing fine. This one is ready to merge.

**Pull request merged**